types/evaluatex/dist/evaluatex.d.ts

```diff
@@ -0,0 +1,2 @@
+export * from '../index';
```

Comment:
so this file (aka module) hosts and export this func, you can write that way to better reflect sources of types (declared in 'evaluatex', re-exported in the index):
```ts
import type { EvaluatexResult } from './../';

declare function evaluatex(
    /** an ASCII or LaTeX expression to be parsed and evaluated. */
    expression: string,
    /** a map of constant values - values that don't change if you invoke fn more than once. */
    constants?: Record<string, number>,
    /** a map of options for the compiler. */
    options?: { latex?: boolean },
): EvaluatexResult;

export default evaluatex;
```
(index):
```ts
import _ from './dist/evaluatex';

declare namespace evaluatex {
    type Token =
        | {
              type: 'NUMBER' | 'POWER' | 'DIVIDE' | 'LPAREN' | 'RPAREN' | 'COMMAND';
              value: string | number;
              name: string | null;
          }
        | {
              type: 'COMMAND';
              value(params: unknown[]): unknown;
              name: string | null;
          }
        | { type: 'SYMBOL'; value: string; name: null };

    interface EvaluatexResult {
        /**
         * @param variables a map of variables that can change between invocations of fn.
         * @returns the numerical result of the calculation.
         */
        (variables?: Record<string, number>): number;
        tokens: Token[];
        expression: string;
    }
}

declare const evaluatex: typeof _;

export = evaluatex;
```